pkg/cmdutil/execute.go

```diff
@@ -76,6 +76,12 @@ func ExecuteCommand(cmd *cobra.Command) ExitCode {
 			result = multierror.Append(result, ErrCommandTimeout)
 		}
 
+		// if error is ErrNothingToUpgrade, exit with zero code
+		if errors.Is(err, ErrNothingToPrepare) {
+			fmt.Println(err.Error())
```

Comment:
doesn't this print to stdout? Shouldn't it print to stderr?

Reply:
Good point! I'll fix that.